testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/saml/CreateAuthnRequestStepBuilder.java

```diff
@@ -79,9 +79,10 @@ public CreateAuthnRequestStepBuilder relayState(String relayState) {
         return this;
     }
 
-    public CreateAuthnRequestStepBuilder signWith(String signingPrivateKeyPem, String signingPublicKeyPem) {
+    public CreateAuthnRequestStepBuilder signWith(String signingPrivateKeyPem, String signingPublicKeyPem, String signingCertificate) {
```

Comment:
Changing this method signature leads to many unnecessary changes. There is no need to do so, please revert, as well as related changes in the testsuite. Feel free to create a overloaded version of this method with this signature, but retain the previous version.

Reply:
Thanks for your review! I reverted the unnecessary changes you pointed out and pushed two separate commits, the first with the bare minimum issue fix, the related test and the correct commit message, and a second one that adds the ability to specify the certificate to be used in AuthnRequest/LogoutRequest tests.

Please let me know if they are both good and I'll squash the commits, but if you prefer not touching the other test files I will just revert the second one.